### DISCUSSION TITLE
How to convert 'GraphQLObjectType' to text

### DISCUSSION THREAD

**Discussion by hengyoush** [graphql-java/graphql-java#3227] [Q&A]
I created a GraphQLObjectType programmatically following the instructions in "Creating a schemaprogrammatically" section(https://www.graphql-java.com/documentation/schema/#creating-a-schema-programmatically). But how do I convert the GraphQLObjectType that I have created into text form.
for example:
```java
GraphQLObjectType type = newObject().name("Range")
                .field(newFieldDefinition().name("value").type(list(GraphQLInt)).build())
                .field(newFieldDefinition().name("count").type(list(GraphQLInt)).build())
                .field(newFieldDefinition().name("sum").type(list(GraphQLFloat)).build())
                .build()
```
How to convert the above 'type' to schema text like:
```
type Range {
	value: [Int]
	count : [Int]
	sum: [Float]
}
```

**Comment by bbakerman** [comment]
See `graphql.schema.idl.SchemaPrinter` and `graphql.schema.idl.SchemaPrinter#print(graphql.schema.GraphQLType)` specifically
